Jenkinsfile

```diff
@@ -0,0 +1,53 @@
+pipeline {
+
+    options {
+        timestamps()
+        timeout(time: 45, unit: 'MINUTES', activity: true)
+    }
+
+    agent {
+        node {
+            label "cloud-ccp-ci"
+        }
+    }
+    environment {
+        SOCOK8S_ENVNAME = "cloud-socok8s-${env.BRANCH_NAME}-${env.BUILD_NUMBER}"
+        OS_CLOUD = "engcloud-cloud-ci"
+        KEYNAME = "engcloud-cloud-ci"
+        DELETE_ANYWAY = "YES"
+        SOCOK8S_DEVELOPER_MODE = "True"
+        DEPLOYMENT_MECHANISM = "openstack"
```

Comment:
can we set `ANSIBLE_STDOUT_CALLBACK=yaml` here so the output from the job is more readable?

Reply:
Done. Thanks!